.github/workflows/compressed-size.yml

```diff
@@ -0,0 +1,28 @@
+name: Compressed Size
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+env:
+  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+  TURBO_TEAM: ${{ vars.TURBO_TEAM }}
+  CI: ${{ vars.CI }}
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
@manudeli 
I'm not sure about this code. Is it necessary?"

Reply:
Yes! This concurrency keyword prevent unnecessary workflow run if new commit made during running worflow exist